contracts/base/RegistryManager.sol

```diff
@@ -9,14 +9,14 @@ contract RegistryManager is Ownable2Step {
 
     event RegistryChanged(address indexed oldRegistry, address indexed newRegistry);
 
-    error PluginNotPermitted(address plugin, uint64 listedAt, uint64 flaggedAt);
+    error IntegrationNotPermitted(address plugin, uint64 listedAt, uint64 flaggedAt);
     error AccountDoesNotImplementValidInterfaceId(address account);
 
-    modifier onlyPermittedPlugin(address plugin) {
+    modifier onlyPermittedIntegration(address plugin) {
         // Only allow registered and non-flagged plugins
         (uint64 listedAt, uint64 flaggedAt) = ISafeProtocolRegistry(registry).check(plugin);
```

Comment:
Unrelated to this PR, but could we add a reference to the Rhinestone registry interface/ ERC here?

Reply:
Not sure why to refer it here as the function signature for Rhinestone is different.

https://github.com/rhinestonewtf/registry/blob/29084fa663a22019e7a80a3707ecf746e087af9d/src/interface/IQuery.sol#L23

Reply:
We will mention it in Specs as discussed.